### ISSUE TITLE
How disable download file to default browser folder

### ISSUE THREAD

**Issue by imbaMF** [go-rod/rod#291]
**Rod Version:** v0.81.3

## The code to demonstrate your question
![1](https://user-images.githubusercontent.com/11431176/100322159-866c2400-2fcc-11eb-9492-77b1d4dc9512.png)
![2](https://user-images.githubusercontent.com/11431176/100322164-879d5100-2fcc-11eb-8f5e-a28198f342c4.png)

## What have you tried to solve the question
Im just want save dowloaded source file to my folder, but browser saves file to the default browser folder too.
How to disable it?

**Comment by ysmood** [comment]
The download won't be downloaded to the default folder:

https://github.com/go-rod/rod/blob/684d32090e4bc82a8d0a1aa04c20e3d9330c1822/hijack.go#L378

Can you provide minimal code for us to reproduce your issue?

**Comment by imbaMF** [comment]
Im trying to make example, and im stuck, el.MustClick() did nothing, why?
```
package main

import (
	"time"

	"github.com/go-rod/rod"
	"github.com/go-rod/rod/lib/launcher"
	"github.com/go-rod/rod/lib/utils"
)

func main() {
	ln := launcher.New().Headless(false)

	defer ln.Cleanup()

	ctl := ln.MustLaunch()

	browser := rod.New().
		ControlURL(ctl).
		Trace(true).
		SlowMotion(1 * time.Second).
		MustConnect()

	defer browser.MustClose()

	page := browser.MustPages().First().MustWindowMaximize().MustNavigate("https://www.learningcontainer.com/download/sample-pdf-download-10-mb/")

	page.MustWaitLoad()

	wait := page.MustGetDownloadFile("*.zip")
	page.MustElement(".download-on-click").MustClick()
	_ = wait()

	utils.Sleep(2)
}

```

**Comment by ysmood** [comment]
Seems like a bug, I will fix it later.

**Comment by imbaMF** [comment]
When script open multiple windows at once to download files from this windows, once in about > 24 hours scripts crashes... i cant make minimal example, becouse i dont know when it happens i just can show panic message...
![1](https://user-images.githubusercontent.com/11431176/100426753-1035f300-309a-11eb-8130-3c4c2d9bc2db.png)

**Comment by ysmood** [comment]
Released v0.82.0, the new way to download files:

https://github.com/go-rod/rod/blob/6d2822e6b8301076f3d2b09c5d7e2bd6c1dd99c0/examples_test.go#L458-L466
